packages/user-mgnt/src/features/createUser/service.ts

```diff
@@ -94,6 +94,34 @@ export const getCatchmentAreaIdsByPrimaryOfficeId = async (
   return catchmentAreaIds
 }
 
+export const updateFhir = async (token: string, resource: fhir.Resource) => {
```

Comment:
Let's not duplicate the whole function here, instead let's adjust the existing function `postFhir` to do the work.